### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lukacsja**
[DEMO LINK](https://lukacsja.github.io/react_clock/)

### CODE REVIEW

**Review by Lun4trick — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,39 +1,59 @@
...
+
+  render() {
+    const { clockName, hasClock } = this.state;
+
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        {
+          hasClock && <Clock clockName={clockName} />
+        }
```
Comment (naziamoff) [inline src/App.tsx:55]: looks more clear this way, I guess
```suggestion
        {hasClock && (
          <Clock clockName={clockName} />
        )}
```

File: src/Clock.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+export class Clock extends React.Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const today = new Date();
```
Comment (naziamoff) [inline src/Clock.tsx:21]: basically that is not `today`, but `now`

File: src/Clock.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```
Comment (naziamoff) [inline src/Clock.tsx:1]: better not to disable for the whole document, there's a chance you gonna miss to remove some redundant logs. It is better to disable linter for the specific lines of your code

**Review by naziamoff — APPROVED** [review]
well done, some minor suggestions

**Pull request closed without merging**